riak/transports/tcp/connection.py

```diff
@@ -202,6 +202,7 @@ def _connect(self):
                                                         self._timeout)
             else:
                 self._socket = socket.create_connection(self._address)
+            # logging.debug('[TcpConnection] new connection created')
```

Comment:
Well, it's useful to confirm that multi operations are actually working & creating multiple TCP connections